src/Milkis.purs

```diff
@@ -65,12 +74,8 @@ defaultFetchOptions =
   }
 
 fetch
-  :: forall options trash eff
-   . Union options trash Options
-  => FetchImpl
-  -> URL
-  -> Record (method :: Method | options)
-  -> Aff eff Response
+  :: FetchImpl
+  -> Fetch
```

Comment:
I think it's fine to leave it. No need to alias here, right?

Reply:
but that means we have to change it in two places every time right?

Reply:
tru. I see ur point